lib/Traits/TraitsClasses/CartoMapV3CatalogItemTraits.ts

```diff
@@ -0,0 +1,62 @@
+import primitiveArrayTrait from "../Decorators/primitiveArrayTrait";
+import primitiveTrait from "../Decorators/primitiveTrait";
+import mixTraits from "../mixTraits";
+import { GeoJsonTraits } from "./GeoJsonTraits";
+
+export default class CartoMapV3TableCatalogItemTraits extends mixTraits(
+  GeoJsonTraits
+) {
+  @primitiveTrait({
+    type: "string",
+    name: "Access token",
+    description: "The access token to pass to the Carto Maps API"
+  })
+  accessToken?: string;
+
+  @primitiveTrait({
+    type: "string",
+    name: "Authorization token",
+    description: "The authorization token to pass to the Carto Maps API"
+  })
+  connectionName = "carto_dw";
+
+  @primitiveTrait({
+    type: "string",
+    name: "Base URL",
+    description:
+      'Base URL for Carto API (eg "https://gcp-us-east1.api.carto.com/")'
+  })
+  baseUrl = "https://gcp-us-east1.api.carto.com/";
+
+  @primitiveTrait({
+    type: "string",
+    name: "Geo column name",
```

Comment:
Should be "Geom column name"?

Reply:
The API call to Carto uses `geo_column` - so I would prefer to keep it is close to that as possible.